topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -159,6 +171,10 @@ private void configureApplauncher() {
 		appLauncher.getElement().addClassName("headertabs");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "headertabs" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)